### DISCUSSION TITLE
Toast to the Workspaces Launch 3/12/2024

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#30] [Show and tell]
Hey folks, something we did at GitHub prior to the Microsoft acquisition was a thread celebrating big launches. The last 3 months the team created two new labels to track one of the biggest product updates to date. This could not have been possible without a great community, so I'd love to welcome everyone to commemorate and share a toast with us. 

Read about Workspaces [here](https://opensauced.pizza/blog/scaling-oss-projects) & the [Changelog](https://opensauced.pizza/changelog/workspaces-beta)

## How to toast
This is the fun part. 

1. Take a selfie with a drink
2. Post as a comment below with a shout to someone or the team that made this happen.

**Comment by bdougie** [comment]
![Screen Shot 2024-03-12 at 10 15 12 AM](https://github.com/open-sauced/feedback/assets/5713670/3cd12932-0040-4541-ba31-bb378048f924)

Shout out to the engineers @nickytonline @brandonroberts @jpmcb and @zeucapua for showing up and getting PRs opened and reviewed. 

Shout out to @isabensusan for making sure the trains run on time and getting us organized internally. 

Special shout out the docs team @BekahHW & @adiati98 for making sure we had context and guides for users.

Finally special shout out to the @open-sauced/triage team. Things have been quiet, but super excited get y'all involved in triaging new issues and usability checking. 

We could have not done it with all of you!! 🍕

**Comment by nickytonline** [comment]
![CleanShot 2024-03-12 at 14 19 27](https://github.com/open-sauced/feedback/assets/833231/1c776b1c-bbf4-41cc-b3b3-0e20df933d0a)

Big ups to my fellow engineers @brandonroberts @jpmcb and @zeucapua! We shipped!

A big thank you to @BekahHW for all the thoughtful and constructive feedback. Thanks as well to @adiati98 for being one of our power users and also a maintainer on some of our projects. Your feedback was super valuable. And thanks to both of you for all things docs!

Another big ups to @isabensusan for all the weekday pairings we had on iterating on design and the MVP!

Also, shoutout to @chhristopher for all the behind the scenes work! 🫡

And as always, thanks so much to all our external contributors for reporting issues, creating pull requests and just trying to break things. 🫡🍕

**Comment by jpmcb** [comment]
![fancy selfie](https://github.com/open-sauced/feedback/assets/23109390/202b1aaa-4b5e-491a-ab87-353b7f1e67d6)

Cheers and congrats to the team on a huge workspaces product launch!! 🚀🥡 in my eyes, this represents a big step forward in the product lifecycle, engineering maturity of the codebases, and flexibility going forward!

Shout out and kudos to the other engineers!!

- @brandonroberts: thank you for your steady leadership and guidance to keep all our efforts on track! You've juggled alot of things this release and it's been a pleasure working and growing under your leadership!
- @zeucapua: I've been very, _very_ impressed by your ability to self start, get up and going with our codebases, and dive head first into some gnarly stuff. You've landed alot of big work in a short amount of time and helped us accelerate towards this launch!!
- @nickytonline: your guidance and ownership over our frontend app has been inspiring. Your dedication to quality, efficient, and excellent software has made this release a huge success.

Big, big shoutout to @BekahHW for guiding us through documentation changes and "baking our own pizza" with finding edge cases and discovering some awesome uses for the product. It's been a guiding light for us as we moved towards launch.

👏🏼 for @isabensusan on steering the ship from a product and design perspective!!

And 🥂 to our fearless leader @bdougie - where would any of this be without your vision?!

**Comment by adiati98** [comment]
![ayu with ice macchiato1](https://github.com/open-sauced/feedback/assets/45172775/4d706f01-5de6-48e5-b9ea-fafc8a9ead0a)

Shout out and congratulations to @bdougie, @brandonroberts, @nickytonline, @zeucapua, @jpmcb, @isabensusan for all the hard work and the big product launch! You all rock! 🎉

Shout out and big thanks to @BekahHW for her friendship, mentorship, and guidance in docs and beyond that helps me to grow! 🚀

Lastly, thank you, everyone, for making me part of this big journey and celebration!!! It's an honor!!! ✨

**Comment by BekahHW** [comment]
![Screenshot 2024-03-12 at 3 19 26 PM](https://github.com/open-sauced/feedback/assets/34313413/3d76d0bd-3948-4c19-850a-3c59e7d54e3c)

Shoutout to our team on this launch! I really appreciated seeing such a supportive and cohesive team throughout this process. I feel lucky to be on a team with so many smart and hardworking people.

- @jpmcb thank you for being willing to answer my questions and clarify when I don't get it. I really appreciate your attention to detail and all the docs you've been putting together. It's made it so much easier to follow. #OhCaptainMyCaptain
- @nickytonline thanks for always being willing to jump on a call and talk through issues and clarifications. I always learn so much from our conversations.
- @zeucapua you hit the ground running and haven't stopped. It is incredible to watch all of the things you're doing. Thanks for stepping up and moving things forward.
- @brandonroberts thank you for making sure I'm able to do the work I need to do and making sure the public links were viewable. I can't tell you how much I appreciate you jumping on that issue. 
- @isabensusan it's always great to be able to talk through messaging with you and to see the roadmap for the product. 
- @bdougie thank you for your vision and support as we grow and change.
- @adiati98 thank you, thank you, thank you, for all your help, catching edge cases, for making the docs clearer. You're making this a better project. 
- @chhristopher thank you for always spitballing ideas and finding new ways to think about things.

**Comment by brandonroberts** [comment]
![Photo on 3-12-24 at 2 36 PM #2](https://github.com/open-sauced/feedback/assets/42211/2bd5fc12-9d14-425e-9f77-a1dc553af4b9)

I didn't use a one from Resend this time! 😂 

Big shoutout to the product and engineering teams for reaching this big milestone.

@bdougie: You always have the big ideas and the vision for how we can move OpenSauced forward and I'm glad to be a part of this journey.

@jpmcb: You've crushed through literally every hard challenge we've had including Timescale, rearchitecting our API, and continuing to improve our engineering process.

@nickytonline: You've literally turned big parts of the app around in a short amount of time as well as increasing the quality of our codebase. 

@zeucapua: You've literally been crushing it since day one and I've been super impressed with how you've joined the team and made an immediate impact. 

@BekahHW: Your attention to detail and consistency with research and writing super high quality posts have been super impactful to the product and engineering teams.

@isabensusan: Your eye for the details in the design, and willingness to push the vision forward has been amazing.

@chhristopher: Thanks for always putting in the work even when its behind the scenes to help us keep moving along. The production quality of the OpenSauced content is always top notch.

Also thanks the OpenSauced community for contributions, conversations, and being generally awesome!

🍕

**Comment by FatumaA** [comment]
Congratulations all! 🚀 🎉

Just joined the Triage team excited to help out and push OpenSauced forward 🤩 🍕

**Comment by isabensusan** [comment]
![Photo on 12-03-2024 at 16 57](https://github.com/open-sauced/feedback/assets/27322879/3596b580-f1a3-4cdd-8684-cc4cf6c7dc2c)
🧉 🧉 🧉 🧉 🧉 🧉 

Hey everyone, we shipped!! 🚀🚀🚀🚀 

So excited to see our product grow every day, and all because of the best team I could've hoped for!

Big shout out to our incredible engineers for making it happen:  @brandonroberts for your patience dealing with changes and last minute decisions, @jpmcb for supporting all of my crazy ideas, and @nickytonline for being the best daily sync partner! (and also getting a jealousy inducing tan)

And a special mention to @zeucapua : you are the definition of "hitting the ground running"! I'm amazed by the amount of impact you've managed to make in such short time, even with the amount of chaos around.

Another big shout out to @chhristopher and @BekahHW for keeping our content on point and making us shine, and to @bdougie - the ultimate idea man - for bringing the vision and direction we need to make it happen.

And finally, a big thank you to our community for keeping us honest and helping us find (and squash) those pesky bugs! @adiati98 you are the best power user a team could ever hope for 💯

**Comment by zeucapua** [comment]
![Photo on 3-12-24 at 1 44 PM](https://github.com/open-sauced/feedback/assets/20603494/0e004d85-814c-473c-b39d-933196828cad)

Congratulations to everyone involved on a new big ship!

I've only been here a month and you guys have been so welcoming to me, my code and ideas. I can't thank anyone enough for being supportive. I loved working with each and every one of you in getting workspaces out and running for our users!

Shoutout to the engineering crew on helping me squash my bugs! 
- @brandonroberts for excellent leadership and being there for any problem that arises!
- @jpmcb for helming the API ship and getting new endpoints out as fast as the Flash!
- @nickytonline for being an amazing frontend engineer I love learning from and working with!

And of course where will we all be with @isabensusan organizing everything top down, @chhristopher and @BekahHW's marketing magic (seriously 29 days straight blog posts?!), and @bdougie for your company vision and giving me the opportunity to work here in the first place.

Here's to the workspaces ship and the future of OpenSauced!!! 🍕

Cheers from the big stein glass of strawberry milk <3

**Comment by takanome-dev** [comment]
![Photo on 12-03-2024 at 21 39](https://github.com/open-sauced/feedback/assets/79809121/863f93a8-4613-4cf1-b6af-d2ddd861373d)

Congrats on this huge milestone! 🎉 Thanks to the amazing @open-sauced/staff for their hard work.

Excited to jump back into triaging and contributing! 🚀 So grateful to be part of this awesome community! 🙌
